pkg/stream/environment.go

```diff
@@ -15,9 +15,10 @@ const (
 )
 
 type Environment struct {
-	configuration EnvironmentConfiguration
-	locators      []*locator
-	backOffPolicy func(int) time.Duration
+	configuration           EnvironmentConfiguration
+	locators                []*locator
+	backOffPolicy           func(int) time.Duration
+	locatorSelectSequential bool
```

Comment:
I know this is already merged, but would it have been sensible to have `locatorSelectStratey` as an enum with random and sequential as the possible values? This would make the default behavior more evident

Reply:
Go doesn't have enums, but we can mimic the enum behaviour. Our reasoning to keep as-is was KeepItSimple™, this is not exported/public, and the change only happens in tests.

Reply:
I chatted with @ablease  and got a better understanding - since this is only set in tests I agree it's not worth changing.
Side note, I thought Go has something very enum-ish, though: https://www.sohamkamani.com/golang/enums/?utm_content=cmp-true